Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-archiving.controller.ts

```diff
@@ -107,17 +109,11 @@
                     itSystemUsageService.patchSystem($scope.usageId, payload);
                 }
             }
-            $scope.patchSupplier = (field, value) => {
-                var payload = {};
-                payload[field] = value.id;
-                payload["ArchiveSupplier"] = value.text;
-                itSystemUsageService.patchSystem($scope.usageId, payload);
-            }
 
-            if (systemUsage.SupplierId) {
-                $scope.systemUsage.supplier = {
-                    id: systemUsage.SupplierId,
-                    text: systemUsage.ArchiveSupplier
+            if ($scope.usage.archiveSupplierId && $scope.usage.supplier === undefined) {
```

Comment:
odd with two different properties for the supplier - on the dto.. does not look right.
Also it is not possible to clear the selection so the input field is not working as it should

![image](https://user-images.githubusercontent.com/13116889/183370820-c41e49c6-453c-4de1-8778-f1d973dab635.png)

Reply:
moved the ui state away from the usage dto

Reply:
As it turns out, the missing "x" was caused by the absense of `data-placeholder="Vælg arkiveringsleverandør"`

That isvery odd

Reply:
The select2LoadingService now sets a default placeholder of " ", preventing this from happening